src/confirm.rs

```diff
@@ -68,6 +68,26 @@ async fn transaction_receipt_block_number_check<T: Transport>(eth: &Eth<T>, hash
     Ok(receipt.and_then(|receipt| receipt.block_number))
 }
 
+async fn decode_revert_reason(revert_reason_abi: &str) -> error::Result<String> {
+    let method_id_len = 10;
+    let block_len = 64;
```

Comment:
Please define constants (ideally with references to solidity/ethreum documentation)

Reply:
defined